generate_rss.py

```diff
@@ -131,7 +122,7 @@ def pep_creation(full_path: Path) -> datetime.datetime:
 
 def parse_rst(full_path: Path) -> nodes.document:
     text = full_path.read_text(encoding="utf-8")
-    settings = frontend.OptionParser((rst.Parser,)).get_default_values()
+    settings = docutils.frontend.get_default_settings(rst.Parser)
```

Comment:
this should probably be 

```suggestion
    settings = frontend.get_default_settings(rst.Parser)
```
Or we should add `import docutils.frontend` in the preamble.

A

Reply:
Added `import docutils.frontend`